### ISSUE TITLE
@bugsnag/plugin-react-native-unhandled-rejection crashing on start with react-native-web

### ISSUE THREAD

**Issue by robertherber** [bugsnag/bugsnag-js#1458]
### Describe the bug
I'm getting 
```
Module parse failed: Unexpected token (54:16)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file.
```
with the latest version (`7.10.2`) of `@bugsnag/plugin-react-native-unhandled-rejection` - it works in `7.10.0`.

I guess the `??` expression at line 54 would require transpilation for browser usage - whereas it is handled/transpiled for native.

### Environment
`expo@42.0.0` 
`react-native-web@0.17.1`
`@bugsnag/expo@7.10.4`
`@bugsnag/plugin-react-native-unhandled-rejection@7.10.2` (installed as a dependency)

### Workaround
It works by setting a resolution in `package.json` pointing to the earlier version that's working:
```json
"resolutions": {
    "@bugsnag/plugin-react-native-unhandled-rejection": "7.10.0"
  },
```

**Comment by mattdyoung** [comment]
Hi @robertherber 

Thanks for the report. We'll aim to get this issue fixed in an upcoming release.

Note that we don't yet offer official support for React Native / Expo on web, only on Android and iOS, so even after we fix this regression there may be some behavior not fully supported on web. We do have plans on our roadmap to fully support this in the future though.

**Comment by yousif-bugsnag** [comment]
This has been fixed in in [v7.10.5](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.10.5)
